contracts/NestedFactory.sol

```diff
@@ -138,7 +137,7 @@ contract NestedFactory is INestedFactory, ReentrancyGuard, OwnableProxyDelegatio
     /// @inheritdoc INestedFactory
     function unlockTokens(IERC20 _token) external override onlyOwner {
         uint256 amount = _token.balanceOf(address(this));
-        _token.safeTransfer(owner(), amount);
+        SafeERC20.safeTransfer(_token, msg.sender, amount);
```

Comment:
⚠️ The logic has changed here. Is that on purpose? 
Previous logic was sending tokens to the owner.

Reply:
The function is `onlyOwner`, so `msg.sender = owner()`. It was better to user `msg.sender` instead of calling the function.